### ISSUE TITLE
Segmenatation fault on import

### ISSUE THREAD

**Issue by Mahrkeenerh** [maxbachmann/Levenshtein#12]
I have had no problems installing this library on my windows machine, but when I tried to do it on a raspberry running raspbian, all I got was a seg fault.

`pip3 install Levenshtein`

```
python
import Levenshtein
Segmentation fault
```

**Comment by maxbachmann** [comment]
Hi @Mahrkeenerh I can reproduce the issue. This is an issue in the latest version of https://github.com/maxbachmann/RapidFuzz. So for now you can explicitly install `rapidfuzz==1.5.1`.

The issue does not occur when I build the package on arm from source (only when using the wheel provided by piwheels). I am not sure about the root cause yet.

**Comment by maxbachmann** [comment]
@Mahrkeenerh The segmentation fault was caused by a bug in Cython: https://github.com/cython/cython/issues/4377
It is fixed in rapidfuzz version 1.6.2 by manually calling `np.import_array`. However the import unfortunately still raises an exception, since piwheels mistakenly uses an ABI incompatible version of numpy: https://github.com/cython/cython/issues/4377.

Until this is fixed in piwheels you can either install a version below 1.6.0
```
pip install rapidfuzz==1.5.1
```
or build from source
```
pip install rapidfuzz==1.6.2 --no-binary :all:
```
note that building from source in the raspberry pi takes a couple minutes

**Comment by maxbachmann** [comment]
Since Levenshtein does not use `rapidfuzz.process.cdist` this should work with the newest version of RapidFuzz (1.6.2)
